images.yaml

```diff
@@ -68,17 +68,15 @@
   - sha: 72d816c99cc86b117baf04385020121f71015d7f5f9eba2a59e72532d645d38f
     tag: 7.6.1
 - name: fluent/fluent-bit
-  tags:
-  - sha: 6861ee5ea81fbbacf8802c622d9305930b341acc0800bbf30205b4d74ce2b486
-    tag: 0.14.6
+  patterns:
+  - pattern: '>= 1.[0-9]+.[0-9]+'
 - name: fluent/fluent-bit-0.13-dev
   tags:
   - sha: 0ae482ad3e8d951a66090b968389f2a11b0b16b8a208b5edb4ccb0ca5800b90d
     tag: "0.18"
 - name: fluent/fluentd-kubernetes-daemonset
-  tags:
-  - sha: f9fa60dab5e7a6ae4adbe972a2fc22cf7eaa148df74cde4cd7f57e7019260451
-    tag: "v1.3-debian-cloudwatch"
+  patterns:
+  - pattern: '>= 1.[0-9]+.[0-9]+-debian-cloudwatch-1.0'
```

Comment:
Careful with this one - we use a semver comparison and this might try to tag all pre release versions. Let me check.

Reply:
Maybe I can adjust the pattern to 1.[3-9][0-9]*.[0-9]+-debian-cloudwatch-1.0 ?

Reply:
```
{"caller":"retagger/main.go:50","level":"error","message":"failed to execute command","stack":"[{/home/marcel/golang/src/github.com/giantswarm/retagger/cmd/runner.go:36: } {/home/marcel/golang/src/github.com/giantswarm/retagger/cmd/runner.go:98: } {/home/marcel/golang/src/github.com/giantswarm/retagger/pkg/retagger/executor.go:17: } {/home/marcel/golang/src/github.com/giantswarm/retagger/pkg/retagger/job_pattern.go:46: } {/home/marcel/golang/src/github.com/giantswarm/retagger/pkg/retagger/job_pattern.go:105: } {improper constraint: \u003e= 1.[0-9]+.[0-9]+}]","time":"2020-04-08T08:06:38.962915+00:00"}
```
I don't think semver likes this :(

Reply:
Ok, so it is not a regex pattern. I guess 1.x.x would be better then